.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 coverage
 cache
 *.log
+*.out
```

Comment:
What generates these files?

Reply:
I do -- when I'm debugging, I pipe output to out files, like `npm test | tee test.out`